### PR TITLE
added server paths and tests

### PR DESCRIPTION
**Pull request by mattman240**

### CODE REVIEW

File: server/index.test.js [hunk 1]
```diff
@@ -8,3 +8,39 @@ describe('Test the root path', () => {
...
+      } else {
+        done();
+      }
+    });
+});
+
+test('responds with 404 error code for bad GET requests', (done) => {
+  request(app)
+    .get('/somethingRandom/101')
+    .expect(404).end((err) => {
```
Comment (ZeMunchkin) [inline server/index.test.js:27]: I'd make the format more consistent by having .end on another line

File: server/index.test.js [hunk 2]
```diff
@@ -8,3 +8,39 @@ describe('Test the root path', () => {
...
+      } else {
+        done();
+      }
+    });
+});
+
+test('doesn\'t accept post requests', (done) => {
+  request(app)
+    .post('/pictures/101')
+    .expect(404).end((err) => {
```
Comment (ZeMunchkin) [inline server/index.test.js:39]: same thing here

**Review by ZeMunchkin — COMMENTED** [review]
Two formatting changes, and it looks good. :)

File: server/index.js [hunk 3]
```diff
@@ -1,11 +1,13 @@
+const items = require('../database-mongo');
 const express = require('express');
 const bodyParser = require('body-parser');
```
Comment (MichaelDurfey) [inline server/index.js:3]: Is there a reason you'd need bodyParser here?

**Review by MichaelDurfey — APPROVED** [review]
Just one question on your server setup. Looks good though!

**Pull request merged**